nodejs/week2/src/backend/routes/randommeal.js

```diff
@@ -0,0 +1,13 @@
+
+const express = require("express");
+const meal = require("../data/meals.json");
+const router = express.Router();
+
+
+router.get("/", (req, res)=>{
+    const randommeal = meals[Math.floor(Math.random() * meals.length)];
```

Comment:
avoid using complex math as an array index. Put the math in a separate variable.